client/.env

```diff
@@ -1 +1,2 @@
-BROWSER=none
\ No newline at end of file
+BROWSER=none
+DANGEROUSLY_DISABLE_HOST_CHECK=true
```

Comment:
I have to ask...

Reply:
As described in the pr commit message, this is unfortunately necessary to enable Create React App to actually render the page, otherwise you just get an `Invalid Host Header` entry.

Normally, you'd manually set the `HOST` value to avoid this. But since GitPod generates a unique one for each workspace we can't do that.

See: https://stackoverflow.com/a/60250859

Reply:
Thanks, I didn't notice the commit message - reviewed the PR as a whole.